babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -505,13 +507,56 @@ SqlCreate SqlCreateTable(Span s, boolean replace) :
     |
         { query = null; }
     )
+    (
+       index = SqlCreateTableIndex(s)
+       {
+           if (index instanceof SqlPrimaryIndex) {
+               primaryIndex = (SqlPrimaryIndex) index;
+           }
+           //code for non-primary index support can be added here
```

Comment:
nit: We can probably remove this comment as I think the code makes this pretty clear.